lib/daisy_ui_components/button.ex

```diff
@@ -41,6 +43,8 @@ defmodule DaisyUIComponents.Button do
           "btn",
           btn_color(assigns[:color]),
           maybe_add_class(assigns[:ghost], "btn-ghost"),
+          maybe_add_class(assigns[:soft], "btn-soft"),
```

Comment:
could you add a simple test for the new classes? On button_test have a few examples

Reply:
Done